app/views/quota.html

```diff
@@ -6,18 +6,19 @@
       <div class="middle-container">
         <div class="middle-header">
           <div class="container-fluid">
-            <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
```

Comment:
If you remove the breadcrumbs from the view, we should also take the breadcrumbs out of the controller.